### ISSUE TITLE
Support for DD_SERVICE_MAPPING

### ISSUE THREAD

**Issue by hasan-ally** [DataDog/dd-trace-js#1173]
We have been using it in our Java services to collect all traces from a single microservice in a single datadog service.
It will help us if you can support this for NodeJs projects too

**Comment by rochdev** [comment]
We are planning to add this feature soon. In the meantime, service names of individual plugins can be configured programmatically:

```js
// make mysql spans use "my-service" instead of "my-service-mysql" as the service
tracer.use('mysql', 'my-service')
```

This configuration option is available on every [plugin](https://datadoghq.dev/dd-trace-js/modules/plugins.html).

**Comment by mintuhouse** [comment]
@rochdev I have given a shot at adding support for this in #1175. 
Let me know if it looks good or if you want me to make any changes to it.
We are currently integrating datadog across company and having this option will greatly reduce the boilerplate code.

**Comment by bigkraig** [comment]
Thanks for this, the service naming has never made sense to me. Rather than having to have our teams set this for every possible plugin that gets loaded, could we have a single flag to clean up the name for all plugins?

**Comment by rochdev** [comment]
This will be implemented soon similar to how it works in Java.

**Comment by hong823** [comment]
@rochdev do you have any news on this request? Currently, we also have the need to group nodejs plugins into the original service.

**Comment by rochdev** [comment]
@hong823 Will land in the next release after https://github.com/DataDog/dd-trace-js/pull/1417 is merged.

**Comment by IGassmann** [comment]
`DD_SERVICE_MAPPING` is released. However, the mapping of the `aws-sdk` doesn't seem to work properly.

I've set `DD_SERVICE_MAPPING="aws-sdk:my-service"` but traces are still showing up with their default name: `my-service-aws-s3`

**Comment by rochdev** [comment]
@IGassmann Fix will be in the next release early next week.

**Comment by rochdev** [comment]
Closing as this was landed in 0.36 and `aws-sdk` was fixed in 1.0.0.
